app/src/main/java/co/condorlabs/customcomponents/graphics/barsgraph/StackedBarsGraph.kt

```diff
@@ -0,0 +1,270 @@
+                            Color.GRAY
+                        ),
+                        BarSection(
+                            12,
+                            Color.BLUE
+                        )
+                    )
+                ),
+                StackedBar(
+                    "TWO", listOf(
+                        BarSection(
+                            38,
+                            Color.DKGRAY
+                        )
+                    )
+                ),
+                StackedBar(
+                    "THREE", listOf(
+                        BarSection(
+                            7,
+                            Color.RED
+                        ),
+                        BarSection(
+                            2,
+                            Color.BLACK
+                        )
+                    )
+                )
+            )
+        )
+
+    init {
+        attrs?.let { setupAttrs(it) }
+    }
+
+    private fun setupAttrs(attrs: AttributeSet) {
+        val typedArray = context.obtainStyledAttributes(
+            attrs,
+            R.styleable.StackedBarsGraph,
+            DEFAULT_STYLE_ATTR,
+            DEFAULT_STYLE_RES
+        )
+        numbersTextSize =
+            typedArray.getDimension(R.styleable.StackedBarsGraph_stackedBarsGraphNumbersTextSize, defaultTextSize)
+        labelsTextSize =
+            typedArray.getDimension(R.styleable.StackedBarsGraph_stackedBarsGraphLabelsTextSize, defaultTextSize)
+        horizontalLinesStrokeWidth = typedArray.getDimension(
+            R.styleable.StackedBarsGraph_stackedBarsGraphHorizontalLinesStrokeWidth,
+            defaultHorizontalLinesStrokeWidth
+        )
+        barsStrokeWidth = typedArray.getDimension(
+            R.styleable.StackedBarsGraph_stackedBarsGraphBarsStrokeWidth,
+            defaultBarsStrokeWidth
+        )
+        barsMargin =
+            typedArray.getDimension(R.styleable.StackedBarsGraph_stackedBarsGraphBarsMargin, defaultBarsMargin)
+        val numbersColor =
+            typedArray.getColor(R.styleable.StackedBarsGraph_stackedBarsGraphNumbersColor, defaultTextColor)
+        val labelsColor =
+            typedArray.getColor(R.styleable.StackedBarsGraph_stackedBarsGraphLabelsColor, defaultTextColor)
+        val horizontalLinesColor = typedArray.getColor(
+            R.styleable.StackedBarsGraph_stackedBarsGraphHorizontalLinesColor,
+            defaultHorizontalLinesColor
+        )
+        typedArray.recycle()
+
+        numbersPaint.run { textSize = numbersTextSize; color = numbersColor }
+        labelsPaint.run { textSize = labelsTextSize; color = labelsColor }
+        horizontalLinesPaint.run {
+            strokeWidth = horizontalLinesStrokeWidth; color = horizontalLinesColor
+        }
+        barsPaint.strokeWidth = barsStrokeWidth
+        setupConfig(stackedBarsGraphConfigOfExample)
+    }
+
+    fun setupConfig(stackedBarsGraphConfig: StackedBarsGraphConfig) {
```

Comment:
Can be private.

Reply:
never and ever!
This method allows assigning the object with the information to graph.